x-pack/legacy/plugins/integrations_manager/public/components/content_collapse.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useState, useRef, useLayoutEffect, Fragment, useCallback } from 'react';
+import { EuiButtonEmpty, EuiSpacer, EuiButton, EuiHorizontalRule } from '@elastic/eui';
+import euiStyled from '../../../../common/eui_styled_components';
+
+const BottomFade = euiStyled.div`
+width: 100%;
+background: ${props =>
+  `linear-gradient(rgba(255, 255, 255, 0) 0%, ${props.theme.eui.euiColorGhost} 100%)`};
```

Comment:
The current gradient looks odd to me in dark mode. I adjusted to this a

```suggestion
  `linear-gradient(${props.theme.eui.euiColorEmptyShade}00 0%, ${props.theme.eui.euiColorEmptyShade} 100%)`};
```

 * Uses `euiColorEmptyShade` so it'll use the desired color in dark mode as well.
 * Uses [8 digit hex](https://drafts.csswg.org/css-color/#hex-notation) `${props.theme.eui.euiColorEmptyShade}00`(e.g. `#ffffff00`) so we can use the EUI color vars which are in hex. I'm not sure what our CSS post processing setup is but the 8 digit syntax [isn't supported in IE 11](https://caniuse.com/#feat=css-rrggbbaa) so we might need to add a plugin like https://github.com/postcss/postcss-color-hex-alpha. Or perhaps we have SASS or other color utils to create an `rgba()` value from a hex code

Either approach will have a challenge when use on a background other than black or white, but we can address that later.

####  PR in dark mode
<img width="655" alt="Screen Shot 2019-11-12 at 9 10 28 AM" src="https://user-images.githubusercontent.com/57655/68678634-4efb1880-052c-11ea-83f7-a95cf1e80fee.png">

#### Suggestion in dark theme
<img width="661" alt="Screen Shot 2019-11-12 at 9 09 47 AM" src="https://user-images.githubusercontent.com/57655/68678667-5fab8e80-052c-11ea-81a5-dd4abff76136.png">

#### PR & suggestion in light theme
<img width="667" alt="Screen Shot 2019-11-12 at 9 06 44 AM" src="https://user-images.githubusercontent.com/57655/68678668-60442500-052c-11ea-8b32-3339ca1b211d.png">

#### PR & suggestion in light theme against different container `background-color`
<img width="687" alt="Screen Shot 2019-11-12 at 9 06 14 AM" src="https://user-images.githubusercontent.com/57655/68678669-60442500-052c-11ea-9659-f35d6825838f.png">

Reply:
Good catch.  I could check if we're in darkMode and then use the darker color but i'd be doing more hardcoding though it would work in IE.  I think this 8 digit hex solution is best and have no fade in IE11 for now.